apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -318,47 +328,50 @@ do_select(
     Node,
     QueryState0 = #{
         table := Tab,
-        mactch_spec := Ms,
-        fuzzy_fun := FuzzyFun,
-        continuation := Continuation,
-        limit := Limit
+        match_spec := Ms,
+        limit := Limit,
+        complete := false
     }
 ) ->
     QueryState = maybe_apply_total_query(Node, QueryState0),
     Result =
-        case Continuation of
-            ?FRESH_SELECT ->
+        case maps:get(continuation, QueryState, undefined) of
+            undefined ->
                 ets:select(Tab, Ms, Limit);
-            _ ->
+            Continuation ->
                 %% XXX: Repair is necessary because we pass Continuation back
                 %% and forth through the nodes in the `do_cluster_query`
                 ets:select(ets:repair_continuation(Continuation, Ms))
         end,
     case Result of
-        '$end_of_table' ->
-            {[], QueryState#{continuation => ?FRESH_SELECT}};
+        {Rows, '$end_of_table'} ->
```

Comment:
`ets:continuation()` says it's an opaque value.  Is it safe to match its value here?

Reply:
Heh, [the code says](https://github.com/erlang/otp/blob/6edaed6e23779057965231abd4e4b0421d566ec7/lib/stdlib/src/ets.erl#L52-L54) it's not opaque (as opposed to the docs). Also, dialyzer seems to be happy with this line when run manually.

Reply:
I think the docs say it's opaque because, although it's defined in the module, it's not exported.